tox.ini

```diff
@@ -32,10 +32,8 @@ deps = mypy
 [testenv:bundle]
```

Comment:
Do we need this file at all ?

Reply:
Indeed. Removed. Would be nice to also present how to run in prod. But better to do it with an example.